src/utils/cli_options.jl

```diff
@@ -71,8 +71,9 @@ function argparse_settings()
         arg_type = String
         default = "false"
         "--surface_scheme"
-        help = "Surface flux scheme [`nothing` (default), `bulk`, `monin_obukhov`]"
+        help = "Surface flux scheme [`BulkSurface` (default), `MoninObukhovSurface`]"
```

Comment:
All the paper based options are not mentioned here (which I agree with). There is no surface scheme `Bomex`